AttitudeManager/Tests/AMTest.cpp

```diff
@@ -0,0 +1,295 @@
+#include <gtest/gtest.h>
+#include <gmock/gmock.h>
+#include "AM.hpp"
+#include "config.hpp"
+
+namespace AM {
+
+TEST(AttitudeManager, MotorInitializationAndOutput) {
+
+    // No motors
+    {
+        config::NUM_MOTORS = 0;
+        config::motors = new config::Motor_t[config::NUM_MOTORS];
+
+        AM::Flightmode *control_algorithm  = config::flightmodes[0].flightmodeConstructor();
+        AM::AttitudeManager am{control_algorithm};
+
+        // Non null references
+        EXPECT_THAT(am.motorReferences_[0], ::testing::NotNull());
+        EXPECT_THAT(am.motorReferences_[1], ::testing::NotNull());
+        EXPECT_THAT(am.motorReferences_[2], ::testing::NotNull());
+        EXPECT_THAT(am.motorReferences_[3], ::testing::NotNull());
+        EXPECT_THAT(am.motorReferences_[4], ::testing::NotNull());
+
+        // Shouldn't do anything
+        am.outputToMotor(config::throttle, 40);
+        am.outputToMotor(config::pitch, 30);
+        am.outputToMotor(config::roll, 20);
+        am.outputToMotor(config::yaw, 10);
```

Comment:
I don't think I can as there are no values to edit in the first place, this was more so to ensure it doesn't crash or segfault, not sure if there is a more official way to indicate that that's what I'm testing for

Reply:
Hey so I still have this test in there, should I? or I guess is there a better way to indicate I am testing for the function to not segfault. I can't really use EXPECT_CALL as I did in other tests because there is no MockChannel I can look at when there are no motors